src/Resources/config/routing.yml

```diff
@@ -1,3 +1,4 @@
+# NEXT_MAJOR: remove this file and add upgrade note
```

Comment:
Shouldn't we update the documentation at [`README.md`](https://github.com/sonata-project/EntityAuditBundle/tree/d68de63687b49d4859ae233ab90bc40e7b089add#appconfigroutingyml) in order to use the routes from the XML file?

Reply:
Done.